scripts/test/Muon/load_run_widget/loadrun_presenter_current_run_test.py

```diff
@@ -94,16 +111,18 @@ def test_load_current_run_correctly_displays_run_if_load_successful(self):
 
         self.assertEqual(self.view.get_run_edit_text(), '1234')
 
-    def test_load_current_run_displays_error_message_if_fails_to_load(self):
+    def test_load_current_run_emits_error_signal_if_fails_to_load(self):
+        self.create_mock_signal_handler()
         self.load_utils_patcher.load_workspace_from_filename = mock.Mock(side_effect=self.load_failure)
 
         self.presenter.handle_load_current_run()
         self.wait_for_thread(self.presenter._load_thread)
 
-        self.assertEqual(self.view.warning_popup.call_count, 1)
+        self.assertEqual(self.signal_handler.call_count, 1)
 
     @run_test_with_and_without_threading
-    def test_load_current_run_reverts_to_previous_data_if_fails_to_load(self):
+    @patch("Muon.GUI.Common.load_run_widget.load_run_presenter.thread_model.warning")
+    def test_load_current_run_reverts_to_previous_data_if_fails_to_load(self, warning_mock):
```

Comment:
this test does not use your patch

Reply:
No, for whatever reason the warning popup originates from the `load_run_widget_presenter` in these cases which is already mocked so the patch is not required.

Reply:
could we delete the patch?

Reply:
I think the patch is still required for the tests where it is used, since it is mocking the warning originating from `thread_load_model`

Reply:
could we add a comment explaining that?